### PR TITLE
fix: type fixes and suggest use setTokens over add

### PR DESCRIPTION
**Pull request by shuowu**
* fixes type  for `browser.getIdToken` and `browser.getAccessToken`
* adds warning message for tokenManager.add to use seTokens
* updates readme and test app
* fixes test script for samples

### CODE REVIEW

File: env/index.js [hunk 1]
```diff
@@ -1,7 +1,7 @@
 const dotenv = require('dotenv');
 const fs = require('fs');
 const path = require('path');
-const ROOT_DIR = path.resolve(__dirname, '..', '..');
+const ROOT_DIR = path.resolve(__dirname, '..');
```
Comment (aarongranick-okta) [inline env/index.js:4]: is there expected to be a second `testenv' file in the 'test' directory? Or is this the new location? Our convention is that it is always in the repo root.

Comment (aarongranick-okta) [reply env/index.js:4]: n/m I see this file was moved to env.

File: env/package.json [hunk 2]
```diff
@@ -0,0 +1,6 @@
+{
+  "name": "@okta/env",
+  "version": "1.0.0",
+  "main": "index.js",
+  "private": true
+}
```
Comment (aarongranick-okta) [inline env/package.json:6]: should add devDependency for "dotenv"

Comment (aarongranick-okta) [reply env/package.json:6]: actually should be regular dependency. Then the eslint for extraneous require will not be needed.

**Review by swiftone — APPROVED** [review]

**Pull request merged**